playbooks/roles/ocp-config/templates/install-config.yaml.j2

```diff
@@ -22,5 +22,18 @@ networking:
   - 172.30.0.0/16
 platform:
   none: {}
-pullSecret: '{{ install_config.pull_secret }}'
 sshKey: '{{ install_config.public_ssh_key }}'
+{% if enable_local_registry %}
+pullSecret: '{{ lookup('file', '~/.openshift/pull-secret-updated') | from_json | to_json }}'
```

Comment:
Could you mention in the README that `~/.openshift/pull-secret-updated` and `/opt/registry/certs/domain.crt` are required to be present.